mmv1/third_party/terraform/tests/resource_notebooks_runtime_container_test.go.erb

```diff
@@ -0,0 +1,69 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' %>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
you don't need these tests as separate entities if you already include them as examples. Tests are generated automatically from examples

Reply:
Ah, I was wondering about that. Thought they were really similar files. Deleting them.